VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddPostFragment.java

```diff
@@ -161,6 +161,7 @@ public void onClick(View v) {
                 threadBody.setRawBody(body);
                 threadBody.setTopicId(((DiscussionTopicDepth) topicsSpinner.getSelectedItem()).getDiscussionTopic().getIdentifier());
                 threadBody.setType(discussionQuestion);
+                threadBody.setFollowing(true);  // MA-1380
```

Comment:
I think it'll be better if I just comment it here, cuz default assignments are hard to track as compared to finding the calls to a setter.